drivers/hmis_external_apis/app/models/hmis_external_apis/mci_payload.rb

```diff
@@ -0,0 +1,144 @@
+            'cityCouncilDistrictCode' => 5,
+            'countyCode' => 7,
+            'countyCouncilDistrictCode' => 8,
+            'isCurrentResidence' => true,
+            'isVerified' => true,
+            'municipalityCode' => 19,
+            'schoolDistrictCode' => '12',
+            'stateCode' => 'AZ',
+            'zipCode' => '11111',
+            'zipCodeExt' => 'NDZC',
+            'countryCode' => 97,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+          {
+            'addressLine1' => '423 Broadway Mailing',
+            'addressLine2' => 'unit 3',
+            'addressTypeCode' => 2,
+            'aptNumber' => '102',
+            'city' => 'Cool',
+            'cityCouncilDistrictCode' => 2,
+            'countyCode' => 2,
+            'countyCouncilDistrictCode' => 2,
+            'isCurrentResidence' => true,
+            'isVerified' => true,
+            'municipalityCode' => 2,
+            'schoolDistrictCode' => '2',
+            'stateCode' => 'AZ',
+            'zipCode' => '66212',
+            'zipCodeExt' => '1234',
+            'countryCode' => 37,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+          {
+            'addressLine1' => '515 High Street OTHERU',
+            'addressLine2' => 'Near Opera House OTHERU',
+            'addressTypeCode' => 3,
+            'aptNumber' => '1093',
+            'city' => 'Cool',
+            'cityCouncilDistrictCode' => 3,
+            'countyCode' => 3,
+            'countyCouncilDistrictCode' => 3,
+            'isCurrentResidence' => true,
+            'isVerified' => true,
+            'municipalityCode' => 3,
+            'schoolDistrictCode' => '3',
+            'stateCode' => 'NY',
+            'zipCode' => 'NYZ',
+            'zipCodeExt' => 'NYZE',
+            'countryCode' => 90,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+        ],
+        'contact' => {
+          'homePhone' => '5655652377',
+          'faxNumber' => '5655652504',
+          'email' => 'contacttest1@gmail.com',
+          'workPhone' => '565565',
+          'workPhoneExt' => '33333',
+          'cellPhone' => '7045652777',
+        },
+        'residencyCode' => 3,
+        'firstName' => client.first_name,
+        'middleName' => client.middle_name,
+        'lastName' => client.last_name,
+        'suffix' => client.name_suffix,
+        'ssn' => client.ssn,
+        'birthDate' => client.dob.to_s(:db),
+        'raceCodes' => '2-,',
+        'deathDate' => '2019-05-09T11:23:31.129Z',
+      }
+
+      # if true # FIXME: birth name exists
+      #   payload['birthName'] = {
+      #     'firstName' => client.first_name,
+      #     'middleName' => client.middle_name,
+      #     'lastName' => client.last_name,
+      #     'suffix' => client.name_suffix,
+      #     'dateVerified' => '2019-05-09T11:23:31.127Z',
+      #     'dateUpdated' => '2019-05-09T11:23:31.127Z',
+      #   }
+      # end
```

Comment:
@yakloinsteak this looks great. I can fill in the mappings for race, gender, and ethnicity after this is merged in.
```suggestion
      payload = {
        # 'userID' => 'string',
        'otherNames' => {
          'firstName' => client.preferred_name,
          'lastName' => client.preferred_name,
        },
        # 'isHomeless' => true,
        'ssnAlias' => client.ssn, # Should be last 4 digits of ssn, if we only have that
        'ethnicityCode' => 7, # Gig to do mapping
        'ethnicityDesc' => 'WCDesc', # I would guess not needed, but can pull from ::HudLists.ethnicity_map
        # 'tribeCode' => 414,
        # 'tribeDesc' => 'WCDesc',
        # 'maritalStatus' => 8,
        # 'housingStatus' => 15,
        'genderCode' => 1, # Gig to do mapping
        # 'residencyCode' => 3,
        'firstName' => client.first_name,
        'middleName' => client.middle_name,
        'lastName' => client.last_name,
        'suffix' => client.name_suffix,
        'ssn' => client.ssn, # Should be complete SSN, if we have it
        'birthDate' => client.dob.to_s(:db),
        'raceCodes' => '2-,', # Gig to do mapping
        # 'deathDate' => '2019-05-09T11:23:31.129Z',
      }
```